spec/integration/preview_spec.rb

```diff
@@ -59,6 +59,44 @@
     end
   end
 
+  describe "component batch endpoint (POST /components/batch" do
+
+    describe "content" do
+      before(:each) do
+        post "/components/batch", { :components => [ { :component => id, :options => { :fixture => id } } ] }.to_json
+      end
+
+      let (:response) { JSON.parse(last_response.body.chomp, :symbolize_names => true) }
+
+      context "without a data mapper" do
+        let(:id) { "foo" }
+
+        expected = { :components => [ { :component => "foo", :options => {}, :status => 200, :body => "content\n" } ] }
```

Comment:
Same again, can we multi-line these please as I'd prefer it to be under 80 characters